### ISSUE TITLE
Update documentation / tutorial

### ISSUE THREAD

**Issue by phaazon** [phaazon/luminance#41]

**Comment by YellowOnion** [comment]
It would be nice to have the tutorial updated, as even the provided examples (on github) fail to build now (glVertexID is undeclared?), with modification to the shaders to allow it to compile, I get no picture, and GEDebugger shows that no OpenGL is being used.

I have made some modifications to get this working with SDL2, but I don't believe that is the issue.

**Comment by phaazon** [comment]
Yeah, I’m a bit busy… I’ll integrate your work as soon as I can ;).
